ldp/alg/runners.py

```diff
@@ -298,7 +298,7 @@ class OfflineTrainerConfig(BaseModel):
         ge=1,
     )
     clear_ctx_at_each_iter: bool = False
-    # TODO: add some concept of eval loops
```

Comment:
Removed this comment since, with `num_epochs>1`, one can insert a callback to do eval.